node/src/components/consensus/era_supervisor.rs

```diff
@@ -1116,53 +935,48 @@ where
             }
             ProtocolOutcome::NewEvidence(pub_key) => {
                 info!(%pub_key, era = era_id.value(), "validator equivocated");
-                let mut effects = self
-                    .effect_builder
+                let mut effects = effect_builder
                     .announce_fault_event(era_id, pub_key.clone(), Timestamp::now())
                     .ignore();
-                for e_id in self
-                    .era_supervisor
-                    .iter_future(era_id, self.era_supervisor.bonded_eras())
-                {
-                    let proposed_blocks =
-                        if let Some(era) = self.era_supervisor.active_eras.get_mut(&e_id) {
-                            era.resolve_evidence_and_mark_faulty(&pub_key)
-                        } else {
-                            continue;
-                        };
+                for e_id in self.iter_future(era_id, 1) {
```

Comment:
Here we would use that new constant I left a comment about earlier.

Reply:
Done in https://github.com/casper-network/casper-node/pull/2127/commits/f417ff94706385eda1d3d134cadcc7fec3f6dfd0.